packages/blocks/src/embed-linked-doc-block/embed-linked-doc-block.ts

```diff
@@ -26,18 +26,13 @@ import { SyncedDocErrorIcon } from '../embed-synced-doc-block/styles.js';
 import { styles } from './styles.js';
 import { getEmbedLinkedDocIcons } from './utils.js';
 
-export interface EmbedLinkedDocBlockConfig {
-  handleClick?: (e: MouseEvent, host: EditorHost) => void;
-  handleDoubleClick?: (e: MouseEvent, host: EditorHost) => void;
-}
-
 @customElement('affine-embed-linked-doc-block')
 @Peekable({
   enableOn: ({ doc }: EmbedLinkedDocBlockComponent) => !doc.readonly,
 })
 export class EmbedLinkedDocBlockComponent extends EmbedBlockComponent<
   EmbedLinkedDocModel,
-  EmbedLinkedDocBlockService
+  BlockService
```

Comment:
I think we shouldn't remove the type for service here.

Reply:
Done. Code reveted.